dll/win32/browseui/CAutoComplete.cpp

```diff
@@ -279,6 +280,49 @@ HRESULT WINAPI CAutoComplete::SetOptions(DWORD dwFlag)
     return hr;
 }
 
+/* Edit_BackWord --- Delete previous word in text box */
+static void Edit_BackWord(HWND hwndEdit)
+{
+    INT iStart, iEnd;
+    iStart = iEnd = 0;
+    SendMessageW(hwndEdit, EM_GETSEL, (WPARAM)&iStart, (LPARAM)&iEnd);
+
+    if (iStart != iEnd)
+        return;
+
+    DWORD cchText = GetWindowTextLengthW(hwndEdit);
+    size_t cb = (cchText + 1) * sizeof(WCHAR);
+    LPWSTR pszText = (LPWSTR)CoTaskMemAlloc(cb);
+    if (pszText == NULL)
+        return;
+
+    if (GetWindowTextW(hwndEdit, pszText, cchText + 1) <= 0)
+    {
+        CoTaskMemFree(pszText);
+        return;
+    }
+
+    for (; 0 < iStart; --iStart)
+    {
+        if (pszText[iStart - 1] == L' ' &&
```

Comment:
Just space? Not other whitespace-like character?

Reply:
OK. `GetStringTypeW(CT_CTYPE1, &pszText[iStart - 1], 1, &wType);`

Reply:
Used `shlwapi!IsCharSpacesW`.